x-pack/solutions/security/plugins/security_solution/public/common/components/toolbar/bulk_actions/use_set_alert_tags.tsx

```diff
@@ -68,14 +65,17 @@ export const useSetAlertTags = (): ReturnSetAlertTags => {
           onUpdateFailure(error);
         }
       }
-    };
+    },
+    [ignore, onUpdateFailure, onUpdateSuccess]
+  );
 
-    setAlertTagsRef.current = onSetAlertTags;
+  useEffect(() => {
+    const currentAbortCtrl = abortCtrl.current;
     return (): void => {
-      ignore = true;
-      abortCtrl.abort();
+      setIgnore(true);
```

Comment:
I suspect this ignore stuff is no longer needed at all, and the useState/this line/the conditional checks that are true by default in the onSetAlertTags can all be removed.

Reply:
All removed, thank you for catching this!